src/it/java/com/google/checkstyle/test/base/AbstractPathTestSupport.java

```diff
@@ -44,4 +44,18 @@ protected final String getPath(String filename) throws IOException {
                 .getCanonicalPath();
     }
 
+    /**
+     * Returns canonical path for the file with the given file name.
+     * The path is formed base on the non-compilable resources location.
+     * This implementation uses 'src/test/resources-noncompilable/'
+     * as a non-compilable resource location.
+     * @param filename file name.
+     * @return canonical path for the file with the given file name.
+     * @throws IOException if I/O exception occurs while forming the path.
+     */
+    protected final String getNonCompilablePath(String filename) throws IOException {
```

Comment:
I specifically said `AbstractModuleTestSupport`.
Modules are what requires code to be compilable. Paths are mostly for non-module tests, like CLI output.
IT should mirror test tier as much as possible, eventually they may become the same. This method in the test tier is in `AbstractModuleTestSupport`.

Reply:
done